src/Domain/CMakeLists.txt

```diff
@@ -70,15 +70,14 @@ target_link_libraries(
   DomainBoundaryConditions
```

Comment:
In the elliptic code do you actually evaluate the maps outside of compute items? If not, this commit, if not the previous as well could be replaced by adding a grid->inertial map into the elliptic box and using the time-dependent compute tags we use in the evolution code. Those tags don't do any extra work for "static" domains. I guess they currently require a `Tags::Time` to be in the box, but we could reasonably template on a bool or something to elide the time if it's not needed.

Reply:
Currently the map is only needed during initialization; in multiple actions though (domain init, subdomain init, faces & mortars). I thought about removing the ElementMap tag from the DataBox, but kept it for now so I don't have to recreate the ElementMap in these places. Could be done though I think. When we update the map parameters we have to re-evaluate the map as well though, which might be easier if the ElementMap is available. I haven't thought that part through yet.

Reply:
Alright, that sounds good. Thanks for the explanation!